lib/line/bot/client.rb

```diff
@@ -166,6 +166,19 @@ def verify_id_token(id_token, nonce: nil, user_id: nil)
         post(oauth_endpoint, endpoint_path, payload, headers)
       end
 
+      # Verify access token v2.1
+      #
+      # @param access_token [String] access token
+      #
+      # @return [Net::HTTPResponse]
+      def verify_access_token(access_token)
```

Comment:
https://developers.line.biz/en/reference/line-login-v2/#verify-access-token
There is v2 version of this API.
Should I rename this to `verify_access_token_v2_1`?
I named this way since I thought v2.1 is superset of v2.

Same for `get_profile_by_access_token`.

Reply:
I think `verify_access_token` is fine. Because LINE Login v2.0 is deprecated. Also, the documentation comment says "Verify access token v2.1", so it seems that users will not have to worry about it.